src/matlab2tikz.m

```diff
@@ -5624,29 +5634,23 @@ function printAll(m2t, env, fid)
     versionString = '';
 end
 % ==============================================================================
-function isHG2 = isHG2(m2t)
+function tf = isHG2(m2t)
 % Checks if graphics system is HG2 (true) or HG1 (false).
 % HG1 : MATLAB up to R2014a and currently all OCTAVE versions
 % HG2 : MATLAB starting from R2014b (version 8.4)
-    isHG2 = false;
-    if strcmpi(m2t.env,'MATLAB') && ...
-        ~isVersionBelow(m2t.env, m2t.envVersion, [8,4])
-        isHG2 = true;
-    end
+    tf = strcmpi(m2t.env,'MATLAB') && ...
+        ~isVersionBelow(m2t.env, m2t.envVersion, [8,4]);
 end
 % ==============================================================================
-function isBelow = isVersionBelow(env, versionA, versionB)
+function tf = isVersionBelow(env, versionA, versionB)
```

Comment:
I am following Matlab's convention from the documentation (almost all is* functions have that). I am up for using `bool` if it is clearer for us. I just reckon we should stick with one standard rather than having several different `out` names depending on what the function does. At the end it is a true or false.